### PR TITLE
docs: v0.6.0 release notes

### PR DESCRIPTION
**Pull request by sploiselle**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/5063)

### CODE REVIEW

**Comment by quodlibetor** [comment]
cc #5045

**Review by quodlibetor — APPROVED** [review]

File: doc/user/content/release-notes.md [hunk 1]
```diff
@@ -61,7 +64,7 @@ Wrap your release notes at the 80 character mark.
 - Fix a bug when requesting binary-formatted values with
   [`FETCH`](/sql/fetch/) {{% gh 4976 %}}.
 
-- Fix a bug when using COPY with TAIL that could cause some drivers to
+- Fix a bug when using `COPY` with `TAIL` that could cause some drivers to
   fail if the TAIL was idle for at least 1 second {{% gh 4976 %}}.
```
Comment (msudberg) [inline doc/user/content/release-notes.md:68]: Should TAIL be `TAIL` here as well?

**Comment by awang** [comment]
Can we get the first 2 items documented?  I can't seem to find any other mentions of them outside the release notes:

`Support DEFAULT column values`
and
`Add the version and mz_version functions`

File: doc/user/content/release-notes.md (hunk 1 above)
Comment (sploiselle) [reply doc/user/content/release-notes.md]: Yep, good catch!

**Pull request merged**